### ISSUE TITLE
optimizer initialization

### ISSUE THREAD

**Issue by dkokron** [google-deepmind/graphcast#154]
I am retraining GenCast from scratch using NOAA/GDAS output as training data.  My loss value continued to get smaller through the first epoch and the first half of the second epoch.  Around the middle of the second epoch, the loss jumped up and flat-lined for the remaining half of the second epoch and for all of the third epoch (see attached).  I initialized the optimizer only at the beginning of the training sequence.  I could not reproduce the jump after re-initializing the optimizer (adamw), restarting from a checkpoint and ingested the same input file.

How often did you initialize your optimizer when training?

![Image](https://github.com/user-attachments/assets/f6c00287-b9df-4e42-b9a7-51eb170fb306)
